cts/cts-cli.in

```diff
@@ -139,15 +139,12 @@ def make_test_group(desc, cmd, **kwargs):
     and returned.
 
     The main purpose of this function is to be able to run the same test for
-    both text and XML formats without having to duplicate everything.  Thus, the
-    cmd string may contain "{fmt}", which will have any --output-as= class
-    variable substituted in.
+    both text and XML formats without having to duplicate everything.
     """
     tests = []
 
     for c in [Test, ValidatingTest]:
-        obj = c(desc, apply_substitutions(cmd, extra={"fmt": c.format_args}),
-                **kwargs)
+        obj = c(desc, apply_substitutions(f"{cmd} {c.format_args}"), **kwargs)
```

Comment:
Can split `cmd` on whitespace, insert `c.format_args` after the first item in the list, and then join the items back on a space character into a string.